auto_tests/messenger_test.c

```diff
@@ -232,7 +232,7 @@ START_TEST(test_dht_state_saveloadsave)
      * d) the second save() is of equal content */
     const size_t extra = 64;
     const size_t size = dht_size(m->dht);
-    VLA(uint8_t, buffer, size + 2 * extra);
+    uint8_t* const buffer = malloc((size + 2 * extra) * sizeof(uint8_t));
```

Comment:
Either `uint8_t *const buffer = (uint8_t *)calloc(size ..., sizeof(uint8_t));` or `uint8_t *const buffer = (uint8_t *)malloc(size + 2 * extra);` without the sizeof. If you use sizeof, use calloc.

Same in other places.